ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java

```diff
@@ -208,7 +201,38 @@ private void addAttributesFromPayload(Map<String, String> attributes, ObjectNode
           .ifPresent(v -> attributes.put(Attribute.OS_VERSION, v));
     }
 
+    ParsePayload.addClientIdFromPayload(attributes, json);
```

Comment:
this method doesn't need to be qualified with a class name
```suggestion
    addClientIdFromPayload(attributes, json);
```